webpack/base.config.js

```diff
@@ -30,9 +34,9 @@ module.exports = {
   ],
   module: {
     rules: [
-      // we pass the output from babel loader to react-hot loader
       {
         test: /\.js$/,
+        exclude: /node_modules/,
```

Comment:
what does this change do?

Reply:
`note_modules` is generated by npm/yarn and are already prebuilt, so this just excludes running babel-loader on them in the build process. So, it it should just improve the build speed without affecting anything